.github/workflows/destroy-deployment.yml

```diff
@@ -1,35 +1,50 @@
 name: Destroy-deployments
 
+# When a pull request is merged or just closed, remove deployed docs from the dev server
+# and remove the GitHub deployment with the same name.
+
 on:
   pull_request:
     types:
       - closed
 jobs:
   destroy-deployment:
     runs-on: ubuntu-latest
+    env:
+      # we only remove deployments from the test site, never from prod
+      TARANTOOL_UPDATE_URL: ${{secrets.TARANTOOL_DEVELOP_UPDATE_URL}}
+      TARANTOOL_UPDATE_KEY: ${{secrets.TARANTOOL_UPDATE_KEY}}
+      VKTEAMS_BOT_TOKEN: ${{secrets.VKTEAMS_TARANTOOLBOT_TOKEN}}
     steps:
       - uses: actions/checkout@v2
+        id: checkout
         with:
           token: "${{ secrets.GITHUB_TOKEN }}"
 
       - name: Set branch name from source branch
-        run: echo "BRANCH_NAME=${GITHUB_HEAD_REF##*/}" >> $GITHUB_ENV
+        id: set-branch-name
```

Comment:
why did you add so many id's? how are you going to use them?

Reply:
It's for tarantool/actions/report-job-status, see https://github.com/tarantool/actions/tree/master/report-job-status#prepare-your-workflow